gameflow.md

```diff
@@ -0,0 +1,64 @@
+# Gameflow
+
+## Game Logic
+
+The selected level starts and presents the user a random syllable (Hiragna / Katakana) as well as four possible answers.
```

Comment:
Typo: "Hiragna"

Reply:
Also: by random you mean a randomly selected one within the level right? Maybe you could go even more specific here and mention at least an approximate number of syllables within a level, just to communicate the dimensions. The same goes for the segments down here as well.

Reply:
But in the line below it's mentioned that the available syllables depend on the selected level. Doesn't that explain it already?